RestaurantManagementSystem/RestaurantManagementSystem/src/UseCases/Register/RegisterRequestCommandHandler.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Microsoft.AspNetCore.Http.HttpResults;
+using Microsoft.AspNetCore.Mvc;
+using RestaurantManagementSystem.src.Core.Contracts;
+using RestaurantManagementSystem.src.Core.Entities;
+using RestaurantManagementSystem.src.Infrastructure.Services;
+using shortid.Configuration;
+using shortid;
+
+namespace RestaurantManagementSystem.src.UseCases.Register
+{
+    public class RegisterRequestCommandHandler : IRegisterRequestCommandHandler
+    {
+        private readonly IRequestRestaurant _requestRestaurant;
+        private readonly IEmailService _emailService;
+
+        public RegisterRequestCommandHandler(IRequestRestaurant requestRestaurant, IEmailService emailService)
```

Comment:
too much logic, and configurations here ... think about how to refactor it, it should be 3/4 lines of code. code generation is infrastructure concern